cmd/kubeless/list.go

```diff
@@ -100,26 +100,33 @@ func doList(w io.Writer, tprClient rest.Interface, ns, output string, args []str
 func printFunctions(w io.Writer, functions []*spec.Function, output string) error {
 	if output == "" {
 		table := tablewriter.NewWriter(w)
-		table.SetHeader([]string{"Name", "namespace", "handler", "runtime", "type", "topic", "dependencies"})
+		table.SetHeader([]string{"Name", "description", "namespace", "handler", "runtime", "type", "topic", "dependencies", "memory", "env"})
 		for _, f := range functions {
 			n := f.Metadata.Name
+			desc := f.Spec.Desc
```

Comment:
Right, I mean to say "Description isn't one of the standard annotations, so if it's something we actually want to store, then it needs a prefix".

Fwiw, elsewhere I also suggested just exposing an arbitrary way for the `kubeless` caller to add arbitrary annotations, and if the serverless code wants a description then it can just create its own annotation (ditto for any other arbitrary annotation metadata).  (ie: don't make "description" special)